README.md

```diff
@@ -208,6 +209,7 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
 - United States of America:
   - Federal [EIA](https://www.eia.gov/electricity/data.cfm#gencapacity)
   - States [EIA](https://www.eia.gov/electricity/data/state/)
+  - PJM [PJM](http://www.pjm.com/-/media/markets-ops/ops-analysis/capacity-by-fuel-type-2017.ashx?la=en)
```

Comment:
Move this to the capacity section?

Reply:
- added to zones.json US_PJM... Line 1555 "    "_comment": "Installed capacity in PJM RTO effective June 1, 2017 - http://www.pjm.com/-/media/markets-ops/ops-analysis/capacity-by-fuel-type-2017.ashx?la=en"," 
- but README.md line 212 is under line 120 "### Production capacity data sources" so this is also correct, no?

Reply:
My bad, this line is correct :)